.travis.yml

```diff
@@ -125,6 +125,9 @@ matrix:
         - cargo install-update -a
       script:
         - (cd guide && mdbook build)
+        - cargo doc -p wasm-bindgen --no-deps
+        - cargo doc -p web-sys --no-deps
+        - mv target/doc guide/book/api
```

Comment:
So will the API docs be used be used?

Reply:
Not directly by the book, but I'd find them useful at least to browser a git version of the documentation to checkup on things like the `web-sys` crate and its APIs